packages/utils/src/crypto.ts

```diff
@@ -78,6 +79,14 @@ export function encrypt(params: CryptoTypes.EncryptParams): string {
   return serialize({ type, sealed, iv, senderPublicKey });
 }
 
+export function encodeTypeTwo(message: string): string {
+  const type = encodeTypeByte(TYPE_2);
+  // TODO: serialize iv should be optional
+  const iv = randomBytes(IV_LENGTH);
```

Comment:
`serialize` expects `iv` in its params, so im generating one but its not used. Should i make the param optional?

Reply:
imo better leave a comment that this param is not used as it's required for other types